### PR TITLE
Document spill write stats

### PR DESCRIPTION
**Pull request by duanmeng**

### CODE REVIEW

File: velox/docs/monitoring/stats.rst [hunk 1]
```diff
@@ -69,3 +69,41 @@ These stats are reported only by HashBuild and HashAggregation operators.
...
+These stats are reported by operators that support spilling.
+
+.. list-table::
+   :widths: 50 25 50
+   :header-rows: 1
+
+   * - Stats
+     - Unit
+     - Description
+   * - op.spillFillTime
```
Comment (mbasmanova) [inline velox/docs/monitoring/stats.rst:84]: What is the motivation for adding "op." prefix?

Comment (duanmeng) [reply velox/docs/monitoring/stats.rst:84]: operator-specific stats, follow the convention of hashtable prefix above?

Comment (mbasmanova) [reply velox/docs/monitoring/stats.rst:84]: Let's remove this prefix.

If you want to match "hashtable" prefix, then you need to use "spill" prefix. Notice that "hashtable.xxx" stats are also operator-specific, but they do not have "op." prefix.

Comment (duanmeng) [reply velox/docs/monitoring/stats.rst]: Got it, removed.

**Review by xiaoxmeng — APPROVED** [review]
@duanmeng LGTM. Thanks!

**Pull request closed without merging**